packages/myst-to-tex/src/index.ts

```diff
@@ -296,14 +309,18 @@ class TexSerializer implements ITexSerializer {
   }
 }
 
+function createImportCommands(commands: Set<string>) {
+  return [...commands].sort().map((c) => `\\usepackage{${c}}`);
+}
+
 const plugin: Plugin<[Options?], Root, VFile> = function (opts) {
   this.Compiler = (node, file) => {
     const state = new TexSerializer(file, opts ?? { handlers });
     state.renderChildren(node);
     const tex = (file.result as string).trim();
     const result: LatexResult = {
-      imports: [],
-      commands: [...createMathCommands(state.mathPlugins)],
+      imports: createImportCommands(state.data.imports),
+      commands: createMathCommands(state.data.mathPlugins),
```

Comment:
There are improvements here to make to do this data --> string conversion later after all parts are combined.